taskcluster/ci/signing/kind.yml

```diff
@@ -16,20 +16,29 @@ primary-dependency: build
 
 group-by: build-type
 
+only-for-build-types:
+    - macosx/opt
```

Comment:
I think this will cause the `mac` task to exist as soon as the build lands. I think it would be better to exclude this so we can explicitly enable it at some point after the build.